bigquery_etl/glam/templates/scalar_bucket_counts_v1.sql

```diff
@@ -3,14 +3,28 @@
 {% from 'macros.sql' import enumerate_table_combinations %}
 
 WITH
-{{
-    enumerate_table_combinations(
-        source_table,
-        "all_combos",
-        cubed_attributes,
-        attribute_combinations
-    )
-}},
+build_ids AS (
+  SELECT
+    app_build_id,
+    channel,
+  FROM
+    {{ source_table }}
+  GROUP BY
+    1,
```

Comment:
it's usually good to write the column names explicitly to avoid bugs in case the selected columns get changed